### PR TITLE
Fix #2039: Fix file formatting under ratioinput package

### LINKED ISSUE #2039: Fix file formatting under ratioinput package
**Describe the bug**
All the files, under the package `domain/src/test/java/org/oppia/android/domain/classify/rules/ratioinput` are not in the correct format. 

**Expected behavior**
File format should be follow from this file - [TextInputEqualsRuleClassifierProviderTest](https://github.com/oppia/oppia-android/blob/develop/domain/src/test/java/org/oppia/android/domain/classify/rules/textinput/TextInputEqualsRuleClassifierProviderTest.kt
)

**Correct Order**
Constant values
Dagger injection
Local variables
Test Before setup
Tests
Dagger Setup

### PR DESCRIPTION
**Pull request by MaskedCarrot**
## Explanation

Fixes #2039 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt [hunk 1]
```diff
@@ -87,4 +88,10 @@ object InteractionObjectTestBuilder {
   fun createInt(value: Int): InteractionObject {
     return InteractionObject.newBuilder().setReal(value.toDouble()).build()
   }
+
+  public fun createRatio(value: List<Int>): InteractionObject {
```
Comment (anandwana001) [inline domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt]: As the InteractionObjectTestBuilder is an object, we don't need to specify public here.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Nit change, Thanks @MaskedCarrot

File: domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt (hunk 1 above)
Comment (MaskedCarrot) [reply domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt]: done

**Comment by anandwana001** [comment]
@MaskedCarrot could you put a screenshot on showing all test are passing correctly?

**Comment by MaskedCarrot** [comment]
sure here are the screenshots


`RatioInputEqualsRuleClassifierProviderTest`
Screenshot from 2020-10-29 08-06-01

`RatioInputHasNumberOfTermsEqualToClassifierProviderTest`
Screenshot from 2020-10-29 08-06-25

`RatioInputIsEquivalentRuleClassifierProviderTest`
Screenshot from 2020-10-29 08-06-48

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @MaskedCarrot 

@rt4914 for final pass and merge.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**